### PR TITLE
feat: Responsiveness (Media Queries) (#40329)

### PR DESCRIPTION
**Pull request by Ankush-Negi**

### CODE REVIEW

File: html/css/style.css [hunk 1]
```diff
@@ -58,3 +125,190 @@ ul.breadcrumb li+li:before {
...
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-top: 2%;
+}
+
+@media only screen and (min-width: 992px) {
+    .co-container {
+        margin-left: 100px;
+        margin-right: 100px;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: Make it single

File: html/css/style.css [hunk 2]
```diff
@@ -58,3 +125,190 @@ ul.breadcrumb li+li:before {
...
+
+    .secondrow__second {
+        flex-basis: 40%;
+    }
+}
+
+@media only screen and (max-width: 991px) {
+    .co-container {
+        margin-left: 20px;
+        margin-right: 20px;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: Make it single

File: html/css/style.css (hunk 1 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 2 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

**Review by Visaali-Dentsu — APPROVED** [review]

**Review by VisaaliSuccessive — APPROVED** [review]

**Pull request merged**